Scripts/DCS-BIOS/lib/modules/aircraft_modules/F-4E.lua

```diff
@@ -1452,6 +1452,37 @@ local AIRCRAFT_EFFECTS_DEVICE_ID = 63
 -- Target Designator System (AN-ASQ-153)
 local TARGET_DESIGNATOR_DEVICE_ID = 64
 
+-- Target Designator Set Control
+local TARGET_DESIGNATOR_SET_CONTROL = "Target Designator Set Control"
+
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_RETICLE_BRIGHTNESS", TARGET_DESIGNATOR_DEVICE_ID, 3001, 2733, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "TV Reticle Brightness (black to white)")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_BORESIGHT_AZIMUTH", TARGET_DESIGNATOR_DEVICE_ID, 3002, 2734, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "Boresight Azimuth (12-Vis)")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_BORESIGHT_ELEVATION", TARGET_DESIGNATOR_DEVICE_ID, 3003, 2735, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "Boresight Elevation (12-Vis)")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_BORESIGHT_ROLL", TARGET_DESIGNATOR_DEVICE_ID, 3004, 2736, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "Boresight Roll (9-Vis)")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_STOW", TARGET_DESIGNATOR_DEVICE_ID, 3005, 2754, TARGET_DESIGNATOR_SET_CONTROL, "Stow/Unstow TGP")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_LASER_READY", TARGET_DESIGNATOR_DEVICE_ID, 3006, 2755, TARGET_DESIGNATOR_SET_CONTROL, "Arm/Disarm Laser")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_POWER_ON", TARGET_DESIGNATOR_DEVICE_ID, 3007, 2756, TARGET_DESIGNATOR_SET_CONTROL, "Power On/Off TGP")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_BIT", TARGET_DESIGNATOR_DEVICE_ID, 3008, 2737, TARGET_DESIGNATOR_SET_CONTROL, "Next BIT")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_LIGHT_BRIGHTNESS", TARGET_DESIGNATOR_DEVICE_ID, 3009, 2738, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "Panel Light Brightness")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_REJECT_OVERRIDE", TARGET_DESIGNATOR_DEVICE_ID, 3010, 2739, TARGET_DESIGNATOR_SET_CONTROL, "Reject/Override Range")
+F_4E:defineMultipositionSwitch("WSO_TGT_DESIGNATOR_ACQUISITION", TARGET_DESIGNATOR_DEVICE_ID, 3011, 2740, 3, 0.5, TARGET_DESIGNATOR_SET_CONTROL, "Acquisition Mode")
```

Comment:
```suggestion
F_4E:define3PosTumb0To1("WSO_TGT_DESIGNATOR_ACQUISITION", TARGET_DESIGNATOR_DEVICE_ID, 3011, 2740, 3, TARGET_DESIGNATOR_SET_CONTROL, "Acquisition Mode")
```

Reply:
The 3 can be removed here as well

Reply:
Yes, to many parameters... should I spot it 🤦🏼